lib/auth/firebase_auth.dart

```diff
@@ -11,15 +11,17 @@ class FirebaseAuth {
 
   static FirebaseAuth initialize(String apiKey, TokenStore tokenStore,
       {http.Client? httpClient}) {
-    if (_instance != null) {
+    if (initialized) {
       throw Exception('FirebaseAuth instance was already initialized');
     }
     _instance = FirebaseAuth(apiKey, tokenStore, httpClient: httpClient);
     return _instance!;
   }
 
+  static bool get initialized => _instance != null;
+
   static FirebaseAuth get instance {
-    if (_instance == null) {
+    if (initialized == false) {
```

Comment:
This is a weird notation.
Why not `if (!initialized)`?

Reply:
It's a habit from my software house. `== false` is better to spot than `!`  (simmilar to using `{` even for single `if`)
But if you don't like it - I can change it

Reply:
Your explanation makes sense, but I'd still prefer it changed to match the rest of the code.

I'm honestly surprised it didn't trigger the linter. I guess it may be more common than I thought.

Reply:
It is common and already saved me hours of debugging ;)
But I changed it as you wish.